finance.reckoner.app.yaml

```diff
@@ -0,0 +1,34 @@
+app-id: finance.reckoner.app
+runtime: org.gnome.Platform
+runtime-version: "44"
+sdk: org.gnome.Sdk
+command: reckoner
+separate-locales: false
+finish-args:
+  [
+    --socket=x11,
+    --socket=wayland,
```

Comment:
Does this support native wayland? If yes then use fallback-x11+wayland, if not x11 only.